src/scss/06_components/paragraphs/_image-gallery.scss

```diff
@@ -0,0 +1,38 @@
+.image-gallery {
+  display: grid;
+  gap: $spacing-double;
+  grid-template-columns: repeat(1, 1fr);
+
+  @include breakpoint($breakpoint-s) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include breakpoint($breakpoint-l) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.node--type-page,
+.service--full,
+.unit--full {
```

Comment:
This might need to have node--type-district as well.

Reply:
Updated these!